scripts/DolbyIOManager.cs

```diff
@@ -1,22 +1,50 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 using UnityEngine;
+using Unity.VisualScripting;
 using DolbyIO.Comms;
 
 namespace DolbyIO.Comms.Unity
 {
     [AddComponentMenu("DolbyIO/DolbyIO Manager")]
+    [Inspectable]
     public class DolbyIOManager : MonoBehaviour
     {
         private static DolbyIOSDK _sdk = new DolbyIOSDK();
+        private static List<Action> _backlog = new List<Action>();
```

Comment:
Have you considered using a [ConcurrentQueue<Action>](https://learn.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentqueue-1?view=net-6.0) for this?

Reply:
Yes, but on this one I don't want being able to enqueue while dequeuing, it would block the rendering. I will study a better system after if needed :)